src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -485,6 +485,27 @@ def create_and_verify_containerapp_create_and_update(
             JMESPathCheck('length(@)', 0),
         ])
 
+def create_and_verify_containerapp_create_and_update_env_vars(test_cls, resource_group, name, source_path):
+    # Ensure that the Container App environment is created
+    env_name = test_cls.create_random_name(prefix='env', length=24)
+    create_containerapp_env(test_cls=test_cls, resource_group=resource_group, env_name=env_name)
+
+    # Create the Container App using cloud build
+    test_cls.cmd(f'containerapp create -g {resource_group} -n {name} --environment {env_name} --source \"{source_path}\" --env-vars "testkey1=value1" "testkey2=value2"')
+    app = test_cls.cmd(f'containerapp show -g {resource_group} -n {name}').get_output_in_json()
+
+    # Verify that the Container App has the correct environment variables
+    test_cls.assertEqual(app["properties"]["template"]["containers"][0]["env"], [{"name": "testkey1", "value": "value1"}, {"name": "testkey2", "value": "value2"}])
```

Comment:
You can use `checks=[]` to check the properties

Reply:
```
        self.cmd('containerapp show -g {} -n {}'.format(resource_group, ca_name), checks=[
            JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_MYSQL_HOST`])', 1)
        ])
```

Reply:
Updated.